examples/ITR_UI.py

```diff
@@ -1768,10 +1805,12 @@ def reset_filters(n_clicks_reset):
     if (ProjectionControls.TREND_CALC_METHOD != ITR_median
         or ProjectionControls.LOWER_PERCENTILE != 0.1
         or ProjectionControls.UPPER_PERCENTILE != 0.9
-        or ProjectionControls.TARGET_YEAR != 2050):
+        or ProjectionControls.TARGET_YEAR != 2050
+        or ICompanyData.target_probability != 0.5):
```

Comment:
Here we can see that "one of these things is not like the other", namely lots of ProjectionControls and then one ICompanyData.  There are a few considerations here:

1. How complicated do we want to make `TARGET_PROBABILITY`?  If it only needs to be a global variable that is applied to temperature scoring (and the output of Excel spreadsheets), then `TARGET_PROBABILITY` does not belong in `ICompanyData` at all--it's not a property unique to a company.  But if it is going to be a company-specific number (because we can fine-tune that parameter based on specific company attributes and/or data), then it does belong in company data.  In such a case, the `TARGET_PROBABILITY` value controlled by the GUI would more likely be a default value of some sort of data waterfall, influencing but not absolutely defining the value of per-company probabilities.  If that's true then...
2. As per the dataflow model of the GUI, we need a sequence of company data stages (as-read, updated-by-target-probability) that then feed into temperature scoring.  Again, this need not necessarily affect the data warehouse, unless the data warehouse is housing data that is actually affected by `TARGET_PROBABILITY`.
3. The nexus where either global- or per-company temperature scoring rules should be calculated is `calc_temperature_score`.  If `TARGET_PROBABILITY` is a global value (easiest for initial implementation) then that's where we should test whether `target_prob` is in `changed_ids` and recalculate if so.  If `TARGET_PROBABILITY` merely affects the per-company data, then `calc_temperature_score` needs to be run when upstream company data changes (not only when upstream DataWarehouse changes).